superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -249,17 +252,23 @@ class TabbedSqlEditors extends React.PureComponent {
   }
 
   handleSelect(key) {
-    if (key === 'add_tab') {
+    const qeid = this.props.tabHistory[this.props.tabHistory.length - 1];
```

Comment:
It looks like the means to add a new query tab is nice and clean to me, but this is one in particular that could use a screenshot/gif.

Reply:
In the previous version we used a "fake" tab for adding new tabs. AntD tabs ship this functionality out of the box so we don't need this hack with "add_tab" tab. I'll put some before/after screenshots in PR summary